### PR TITLE
Add option to keep timestamp visible.

### PR DESCRIPTION
**Pull request by juliensoret**
Here is a little modification I suggest.
Original version allows only to display timestamp only when the slider is active. I made small changes to make it possible to keep it displaying all the time.
As it wouldn't be logical to display one timestamp when it's a range. I made the 'range=false' have priority over "alwaysShowDate=true".

Here is a demo = http://kara.62.free.fr/slider_alwaysshowtimestamp/?range=false&alwaysShowDate=true
You can change values of range et alwaysShowDate directly in the URL.

Also, I would like to ask you about your plugin. In the context of a school project, 3 people and I are trying to improve Leaflet and its existing plugins. Yours is interesting and I have some others suggestions to work on. Would you accept if I continue pull requesting changes? (if you want to answer by mail : julien@soret.in ). Danke schön!

### CODE REVIEW

File: SliderControl.js [hunk 1]
```diff
@@ -135,6 +138,9 @@ L.Control.SliderControl = L.Control.extend({
                 }
             }
         });
+        if (!_options.range && _options.alwaysShowDate) {
+            $('#slider-timestamp').html(_options.markers[24].feature.properties.time.substr(0, 19));
```
Comment (juliensoret) [inline SliderControl.js]: I'm not sure why "24" works. But it also works with others smaller lists of points.

**Comment by dwilhelm89** [comment]
You are very welcome to keep sending pull requests! 
Thanks for your input.

**Comment by dwilhelm89** [comment]
If you want to continue working on this project. It would be nice if you would also use grunt to create the minified versions for the updates =)

**Comment by juliensoret** [comment]
About Grunt. I will use it in maximum 1 week, including all the changes that will be merged before.
Security speaking, pull-requesting a gruntified file could make you accept bad injections without you being able to read and detect them, isn't it ?

**Comment by dwilhelm89** [comment]
Never though about that but I guess you're right. It makes it however way more easy to include the library via bower.

**Pull request merged**